ReactWindows/ReactNative.Shared/Views/Text/ReactSpanViewManager.cs

```diff
@@ -110,6 +112,50 @@ public void AddView(DependencyObject parent, DependencyObject child, int index)
 
 #if WINDOWS_UWP
             span.Inlines.Insert(index, inlineChild);
+
+            // Initialize ImportantForAccessibility for the child.
+            // Post on UI thread because the method requires that parent/child
+            // relationship is established but XAML does not set the relationship
+            // until the next loop after the child is added.
+            DispatcherHelpers.RunOnDispatcher(() =>
+            {
+                // Find the UIElement that owns the Span. If the Span has not been added
+                // to an UIElement yet, accessing ElementStart will throw NotSupportedException.
+                DependencyObject spanParent;
+                try
+                {
+                    spanParent = span.ElementStart?.Parent;
+                }
+                catch (NotSupportedException)
+                {
+                    return;
+                }
```

Comment:
When are we expected to hit this? Why would the Span not be added to the UIElement yet?

Reply:
@rigdern Yes. Since the order in which elements are added is not guaranteed, there could be a situation when this is called by the Span has not been added to a parent RichTextBlock. There is no other indication (as far as I could say) but trying to access the Parent property will throw this exception

---
In reply to: [166812759](https://github.com/Microsoft/react-native-windows/pull/1595#discussion_r166812759) [](ancestors = 166812759)